WebApiCoreSeed.WebApi.IntegrationTests/Users/UserIntegrationTest.cs

```diff
@@ -0,0 +1,285 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.EF;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.WebApi.IntegrationTests.Fake;
+using WebApiCoreSeed.WebApi.IntegrationTests.Users.TestData;
+using WebApiCoreSeed.WebApi.Models;
+using Xunit;
+
+namespace WebApiCoreSeed.WebApi.IntegrationTests.Users
+{
+    public class UserIntegrationTest
+    {
+        private const string ResouceUri = "api/users/";
+
+        private readonly TestServer _server;
+        private readonly HttpClient _client;
+
+        private readonly DbContextOptions<WebApiCoreSeedContext> _dbContextOptions;
+
+        public UserIntegrationTest()
+        {
+            FileFaker.Fake();
```

Comment:
Is this really needed?

Reply:
Asking this question because I remember some sorcery required in the version 1.1 but I couldn't find anything like that for the version 2.0 (hoping that they improved this somehow). I guess that is not the case, right?